examples/hello_condor.py

```diff
@@ -0,0 +1,23 @@
+'''
+Requires HTCondor(https://research.cs.wisc.edu/htcondor) to be
+installed & configured on the machine.
+
+You can run this example like this:
+
+    .. code:: console
+
+            $ luigi --module examples.hello_condor examples.HelloCondorTask --local-scheduler
+
+If that does not work, see :ref:`CommandLine`.
```

Comment:
Apologies, the 'HT' is silent ;) (condor DEV joke as it used to be just 'condor' but had to be renamed following a name clash).
Fixed now.